README.md

```diff
@@ -21,7 +22,7 @@ Features:
 
```

Comment:
Worth updating the features list above ^ as most of these have been removed here

Reply:
Good catch :+1: I've pushed some more readme updates